helm-maintainers-onboarding-guide.md

```diff
@@ -149,9 +152,9 @@ We tend to avoid comments of the form "This works fine, but another way of doing
 A PR can only be merged only if:
 
 - All tests are passing for that PR (Circle is green)
-- The PR has one LGTM (small, medium) or two LGTMs (large). The GithUb Review tool is used to LGTM.
-- The PR's milestone is set to the current Minor or Patch release. NEVER, EVER MERGE PRs LABELED `Upcoming - Major`
```

Comment:
I think this statement still rings true. Though instead of it being the `Upcoming - Major` milestone, these PRs are now tagged with the `v4.x` label.

Reply:
Okay, I can reword that. Is "The GithUb (sic) Review tool is used to LGTM" still true? Sometimes I just see comments that say LGTM.

Reply:
A comment or the github review tool is fine. In the past there were multiple review tools. Some maintainers (and community members) used alternative review tools like Reviewable and Gerrit in the past, which made it difficult to track discussions for authors as well as who approved what. This was an explicit callout to not use those tools.